music/src/core/player.ts

```diff
@@ -457,6 +461,37 @@ export class SoundFontPlayer extends BasePlayer {
                       })));
   }
 
+  /**
+   * Loads the audio samples for all valid midi pitches, for a specific program.
+   * **Note**: this method is rather slow: only use it if you're sure
+   * that you need to load _all_ possible samples (for example, you're
+   * playing a stream of live notes from the user) -- otherwise, if you already
+   * have the NoteSequence you have to play, use `loadSamples` instead.
+   *
+   * If you do end up using `loadAllSamples`, make sure you're calling it
+   * asynchronously, as to not block other main thread work (like UI
+   * interactions) while waiting for it to finish:
+   *
+   * @param program (optional) The program to be used for the audio samples.
+   * Default is 0.
+   * @param isDrum (optional) True if drum audio samples should be loaded.
```

Comment:
Does this *only*  load drum samples? If so, specify.

Reply:
ummmm it adds `isDrum: true` to notes, so I think so? I am not sure.